packages/core/src/App/app.jsx

```diff
@@ -54,6 +55,14 @@ const AppWithoutTranslation = ({ root_store }) => {
                 growthbookDecryptionKey: process.env.GROWTHBOOK_DECRYPTION_KEY,
                 rudderstackKey: process.env.RUDDERSTACK_KEY,
             });
+            Analytics.setAttributes({
```

Comment:
This is repeated 2 times, can you make a utility function out of it and then call it once it's needed?

Reply:
Agree, as I was about to duplicate the same comment from earlier.